### PR TITLE
add red hat font to UI

### PR DESCRIPTION
**Pull request by christiemolloy**
Add red hat font across entire UI

Screen Shot 2019-11-25 at 3 56 57 PM

### CODE REVIEW

**Comment by dongniwang** [comment]
Adding @amysueg and @lboehling on this PR so they can take a look as well.

**Comment by amysueg** [comment]
This looks good and thank you! One question I have is what happens with the fonts in the 2 components we include in Syndesis -- Atlasmap (Data Mapper) and Apicurito (for editing APIs). It seems like we need to update to the RH font for those components at the same time, right? Adding @gashcrumb for thoughts.

**Comment by gashcrumb** [comment]
We'll want to check those, I'm not sure what they'll pick up.  I think we could probably merge this now though.

File: app/ui-react/syndesis/src/app/App.css [hunk 1]
```diff
@@ -1,6 +1,3 @@
 @import url('~@syndesis/ui/dist/ui.css');
 @import url('~@syndesis/auto-form/dist/auto-form.css');
-
-body {
-  font-family: "Open Sans", Helvetica, Arial, sans-serif;
-}
+@import url('~@patternfly/patternfly/patternfly-fonts.css');
```
[the remark this answers is not in the document]
Comment (gashcrumb) [reply app/ui-react/syndesis/src/app/App.css]: oh, also you can run `yarn storybook` from within the `packages/ui` directory to work with the storybook for much of Syndesis' UI bits.

**Comment by amysueg** [comment]
@gashcrumb said, "We'll want to check those [Atlasmap & Apicurito], I'm not sure what they'll pick up."
Question for you: What is the process for checking to see whether the Red Hat font is picked up in Atlasmap and Apicurito? If it is not, we will need to open issues for those two embedded apps to use the Red Hat font. Can you please let me know?

@paoloantinori @dlabaj @dongniwang @heiko-braun

**Comment by gashcrumb** [comment]
I was just gonna see how the app looks manually using a couple integrations on staging:

1)  The `TaskAPI-imported-1` integration is an API provider integration, so from the detail page edit the integration and then click "view/edit API definition" at the top-right
2)  The `dm-test` integration has a data mapper step, so again edit the integration, and then click the Configure button on the data mapper step (step #3)

File: app/ui-react/syndesis/src/app/App.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (christiemolloy) [reply app/ui-react/syndesis/src/app/App.css]: Thanks for that @gashcrumb , updated!

**Comment by christiemolloy** [comment]
@gashcrumb @amysueg I updated where the font was being applied, I don't see the fonts being applied on storybook though

**Comment by gashcrumb** [comment]
Yep, probably if having the font applied means updating the `html` element of the page it'd require some storybook configuration, which I'll have to look up.  I'll create an issue to get that done outside of the scope of this PR.

**Comment by gashcrumb** [comment]
Dunno why the bot didn't merge this, did it manually.

**Pull request merged**